web/src/features/menu/list/ListItem.tsx

```diff
@@ -60,6 +60,11 @@ const ListItem = forwardRef<Array<HTMLDivElement | null>, Props>(({ item, index,
             <Text>{item.label}</Text>
             <CustomCheckbox checked={checked}></CustomCheckbox>
           </Flex>
+        ) : item.progress !== undefined ? (
+          <Flex flexDirection="column" w="100%" marginRight="5px">
+            <Text verticalAlign="middle" marginBottom="3px"> {item.label} </Text>
+            <Progress value={item.progress} size="sm" colorScheme={item.colorScheme} borderRadius="md"/>
+          </Flex>
```

Comment:
Maybe instead of adding the progress bar at the bottom and normal text at the top you could try how it's like for side scrolling buttons, where you have smaller faint text at the top and then progress bar at the bottom.

Could look nicer but it may leave some awkward empty space in between 🤔

Reply:
Doesnt look too bad, but for my specific use case I like having it with a heavier font weight.

![image](https://user-images.githubusercontent.com/6852526/205459157-ec9d74c1-a2ae-4311-b5de-9f056e579933.png)

Reply:
Bolder definitely looks better